tachyon/math/polynomials/univariate/univariate_sparse_polynomial_unittest.cc

```diff
@@ -324,6 +323,7 @@ TEST_F(UnivariateSparsePolynomialTest, MulScalar) {
   }
 
   Poly actual = poly * scalar;
+  actual.coefficients().RemoveZeros();
```

Comment:
Isn't `RemoveZeros()` called when multiplication is done? Do we need this code?

Reply:
This test was failing in some random cases and I found that it fails when scalar is 0 the expected is constructed with ill-formed terms and cause the test to fail. I changed to following:

`expected.coefficients().RemoveZeros();`